db/schema.rb

```diff
@@ -36,6 +36,7 @@
     t.jsonb    "form_data"
     t.boolean  "subscribed_member", default: true
     t.boolean  "donation",          default: false
+    t.string   "ip_address"
```

Comment:
It's a pain, but it's from my locally modified schema from work I was doing on another branch. I'm aware of it, and I'll do something about it in my next PR.